### PR TITLE
Using images for animation instead of colour

### PR DESCRIPTION
**Pull request by anand-sundaram**
Check it out and let me know if it looks better than just changing the view's `backgroundColor`.

I am currently using images I cropped from a launchpad image. We can ask our designer to do it more properly, with better colours.

### CODE REVIEW

**Comment by ngzhian** [comment]
LGTM! 
On a slightly related note, do you think we can decouple animation from UIView? I was thinking some sort of `AnimatablePad` protocol or something, where it has 2 methods, `animate(by image: UIImage)`, and `animate(by backgroundColor: UIColor)`, or something along those lines.

**Review by ngzhian — APPROVED** [review]

File: MIDIchlorians/MIDIchlorians/Colour.swift [hunk 1]
```diff
@@ -44,6 +44,28 @@ enum Colour: String {
         }
     }
 
+    var image: UIImageView {
+        var imageView = UIImageView()
+        switch self {
+            case .violet:
+                imageView = UIImageView(image: UIImage(named: "cesiousButton"))
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians/Colour.swift:51]: magic constants here, but fine for now

Comment (anand-sundaram) [reply MIDIchlorians/MIDIchlorians/Colour.swift:51]: Yeah..these are not the final colours. I was hoping we can get our designer to make something similar for more pleasant colours. The ones I have now were cropped from this [image](https://us.novationmusic.com/sites/default/files/resources/image/lp-pro-overhead_0.jpg). Not very important right now.

**Comment by ngzhian** [comment]
Not sure if you still want to change anything in this PR, feel free to merge anytime!  LGTM 👍

**Pull request merged**